src/Microsoft.Azure.SignalR.Common/DefaultBlazorDetector.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Concurrent;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class DefaultBlazorDetector: IBlazorDetector
+    {
+        private readonly ConcurrentDictionary<string, bool> _blazor = new ConcurrentDictionary<string, bool>();
```

Comment:
should it be case insensitive, the hub name as the key?

Reply:
Seems no need. Both set and get use `typeof(THub).Name` which is consistent.